kaldb/src/test/java/com/slack/kaldb/logstore/schema/SchemaAwareLogDocumentBuilderImplTest.java

```diff
@@ -7,16 +7,17 @@
 import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.FieldConflictPolicy.CONVERT_FIELD_VALUE;
 import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.FieldConflictPolicy.DROP_FIELD;
 import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.FieldConflictPolicy.RAISE_ERROR;
-import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.FieldType;
-import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.FieldType.convertFieldValue;
 import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.build;
 import static com.slack.kaldb.logstore.schema.SchemaAwareLogDocumentBuilderImpl.makeNewFieldOfType;
+import static com.slack.kaldb.metadata.schema.FieldType.convertFieldValue;
```

Comment:
Now that this code has been moved the corresponding tests should be moved in the appropriate class as well. We're testing `FieldType` static methods in the `SchemaAwareLogDocumentBuilderImplTest.java` class, when the `SchemaAwareLogDocumentBuilderImpl` no longer contains this code.

Reply:
Moved.